clouddriver-aws/src/main/groovy/com/netflix/spinnaker/clouddriver/aws/provider/agent/ReservationReportCachingAgent.groovy

```diff
@@ -184,8 +210,16 @@ class ReservationReportCachingAgent implements CachingAgent {
         log.info("Fetching reservation report for ${credentials.name}:${region.name}")
 
         def amazonEC2 = amazonClientProvider.getAmazonEC2(credentials, region.name)
-        def reservedInstancesResult = amazonEC2.describeReservedInstances(new DescribeReservedInstancesRequest())
-        reservedInstancesResult.reservedInstances.findAll {
+
+        def cacheView = getCacheView()
+        def reservedInstances = cacheView.getAll(
```

Comment:
There is some chance that on the very first cycle that the `ReservedInstancesCachingAgent` won't have completed yet for each region.

I anticipate pre-seeding the redis (by running the **ONLY** the `ReservedInstancesCachingAgent` locally) and just rolling out as normal.